src/ontodia/customization/props.ts

```diff
@@ -78,6 +79,7 @@ export interface LinkLabelStyle {
             fill?: string;
             'stroke'?: string;
             'stroke-width'?: number;
+            text?: LocalizedString[];
```

Comment:
Why a link label text is a part of **style**? It should be a part of link model.